plugins/scaffolder-react/alpha-api-report.md

```diff
@@ -98,6 +100,9 @@ export type NextFieldExtensionOptions<
   schema?: CustomFieldExtensionSchema;
 };
 
+// @alpha
+export const OngoingTask: (props: TaskPageComponentProps) => JSX.Element;
```

Comment:
Hmm I'm a little torn on adding pages as a whole into the `-react` package.

Normally pages contain routing decisions, which can be pretty painful for when you want to re-use them, especially in our case when we want to show this thing in a different context, and might not want to have that route navigation and we can do some other behaviour instead.

I'm feeling that it would make more sense to expose the `TaskSteps` and the `LogStream` maybe? They feel like they're worth a lot more value when using the building blocks from the `-react` package rather than the entire page?

Reply:
> I'm feeling that it would make more sense to expose the TaskSteps and the LogStream maybe? 

@benjdlambert I think that actually makes a lot of sense.  I'll give that a go and see how I get on.

Reply:
@benjdlambert, that was a good shout. This has made the PR much, much more straightforward than before.  I'm no longer exporting `OngoingTask` and instead exporting the components that it uses.